src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/LanguageServerProjectSystem.cs

```diff
@@ -168,13 +168,28 @@ private async ValueTask LoadOrReloadProjectsAsync(ImmutableSegmentedList<string>
 
         projectBuildManager.StartBatchBuild();
 
+        var displayedToast = 0;
+
         try
         {
             var tasks = new List<Task>();
 
             foreach (var projectPathToLoadOrReload in projectPathsToLoadOrReload)
             {
-                tasks.Add(Task.Run(() => LoadOrReloadProjectAsync(projectPathToLoadOrReload, projectBuildManager, cancellationToken), cancellationToken));
+                tasks.Add(Task.Run(async () =>
+                {
+                    var errorKind = await LoadOrReloadProjectAsync(projectPathToLoadOrReload, projectBuildManager, cancellationToken);
+                    if (errorKind is not null)
+                    {
+                        // We should display a toast when the value of displayedToast is 0.  This will also update the value to 1 meaning we won't send any more toasts.
+                        var shouldShowToast = Interlocked.CompareExchange(ref displayedToast, 1, 0) == 0;
```

Comment:
Not urgent suggestion but learned from experience: used named parameters for the 1 and 0 here. Because if you reverse them...